test/test_examples.sh

```diff
@@ -5,53 +5,63 @@
 
 set -e
 
-WORKSPACE=$1
+SOURCE_DIR=$1
 BUILD_DIR=$2
 INSTALL_DIR=$3
+CMAKE_INSTALL_PREFIX=$4
 
 echo "Running: $0 $*"
 
 function print_usage() {
 	echo "$(basename $0) - test all examples standalone"
-	echo "Usage: $(basename $0) <workspace_dir> <build_dir> <install_dir> <list-of-examples-to-run>"
+	echo "Usage: $(basename $0) <source_dir> <build_dir> <install_dir> <CMAKE_INSTALL_PREFIX> <list-of-examples-to-run>"
 }
 
+# CMAKE_INSTALL_PREFIX can be the empty string ""
 if [ "$3" == "" ]; then
```

Comment:
Does this check mean that only the `source_dir` and `build_dir` parameters are required and all other are optional? I think it would suffice to have a check for the 5th argument here only, with a message like "too few args" on an empty string.

Reply:
Done